.github/workflows/deploy.yml

```diff
@@ -199,48 +199,48 @@ jobs:
     timeout-minutes: 15
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
Is this indentation valid?

Reply:
the indentation is the same for all other `uses:`. I don't see this in my changes?
documentation shows that `uses:` should be indented under steps

prettier changed some double quotes to singles but I don't think that would cause issues?

Reply:
@Jaffacakes82